### PR TITLE
Docu

### PR DESCRIPTION
**Pull request by FriederikeHanssen**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,95 @@
 # awsmegatests
-CloudFormation templates to setup the aws megatests necessary cloud infrastructure
+
+CloudFormation templates to setup the necessary cloud infrastructure
+for aws megatests.
+
+## Process
+
+When a a pull request is made to a pipeline's master branch, a GitHub Action is triggered. It starts an AWS batch job, which has permissions to subsequently spawn more AWS batch jobs. The first batch job pulls the nf-core pipeline from GitHub and starts it. The running pipeline can access a previously created S3 bucket to store any output files, such as the work directory, the trace directory, and the results. The pipeline's progress can be monitored on nf-core's nextflow tower instance. The final results are provided to nf-co.re.
```
[the remark this answers is not in the document]
Comment (FriederikeHanssen) [reply README.md:8]: LGTM!

File: README.md [hunk 2]
```diff
@@ -1,2 +1,95 @@
...
+
+## Steps
+
+This process was setup by following this [guide](https://docs.opendata.aws/genomics-workflows/quick-start/) and customizing a few steps.
+
+### Set up permissions
+
+### (Create an S3 bucket)
+
+There is a central bucket setup for all wor
```
[the remark this answers is not in the document]
Comment (FriederikeHanssen) [reply README.md:24]:
```suggestion
to store the run's intermediate
```

[the remark this answers is not in the document]
Comment (FriederikeHanssen) [reply README.md:24]:
```suggestion
If an S3 bucket does not exist yet, create an S3 bucket to store the run's intermediate files and results. An S3 bucket was created to store all `work` and `results` directories for the AWS tests: `S3:nf-core-awsmegatests`.
```
I think it needs to be `run's`

File: README.md [hunk 3]
```diff
@@ -1,2 +1,95 @@
...
+6. Set 'Existing bucket' to `true`
+7. Set 'Workflow orchestrator' to `Nextflow`
+8. Private subnet -> VPC created on previous step (check on previous step in resources tab), private subnet 1A/2A/3A
+
+### Setup Nextflow resources
+
+
+### Setup GitHub Actions
+In order to set up GitHub Actions, the following script has to be added to the ./github/workflows folder. 
+```
```
[the remark this answers is not in the document]
Comment (FriederikeHanssen) [reply README.md:58]: Yes, that sounds good to me

**Comment by FriederikeHanssen** [comment]
@ggabernet thank you for completing the docu :) . Would propose the either be coherent with **Step 1 **, **Step 2 ** etc or to remove it completely. What do you think?

**Comment by ggabernet** [comment]
Hi @FriederikeHanssen , yes good point! either Steps everywhere or all of them removed, both would work in my opinion

File: README.md [hunk 4]
```diff
@@ -1,3 +1,76 @@
...
+5. Leave the Nextflow container image field empty. You can leave the optional fields as default.
+6. Provide the high priority job queue ARNs generated as output of the previous template.
+7. Acknowledge the capabilities and create stack.
+
+### Step 4: Set up GitHub Actions
+A GitHub Actions workflow example to trigger the AWS tests can be found [here](.github/workflows/awstest.yml). The secrets that it uses need to be set up at an organization level, so that all pipelines can use them:
+
+* AWSTEST_KEY_ID: IAM key ID for the AWS user in the nf-core account organization. A specific user was set with restricted roles to run these tests.
+* AWSTEST_KEY_SECRET: IAM key secret for the same user.
+* AWSTEST_TOWER_TOKEN: token for Nextflow tower, to be able to track all pipeline tests running on AWS.
```
Comment (ggabernet) [inline README.md:70]:
```suggestion
* AWS_ACCESS_KEY_ID: IAM key ID for the AWS user in the nf-core account organization. A specific user was set with restricted roles to run these tests.
* AWS_SECRET_ACCESS_KEY: IAM key secret for the same user.
* AWS_TOWER_TOKEN: token for Nextflow tower, to be able to track all pipeline tests running on AWS.
```

Comment (ggabernet) [reply README.md:70]: one last change, I updated the names yesterday night 😄

**Pull request merged**